pkg/kubelet/qos/policy.go

```diff
@@ -20,6 +20,7 @@ import (
 	v1 "k8s.io/api/core/v1"
```

Comment:
This changes the existing logic, so wouldn’t we need an escape hatch, like checking a FeatureGate of Sidecar Containers? Since Sidecar Containers have already been promoted to beta, and this change will also affect non-Sidecar Containers, I’m not sure if it’s the right choice.

/cc @SergeyKanzhelev

Reply:
Yes, the new logic must be under the sidecar feature gate

Reply:
Done